package.json

```diff
@@ -19,7 +19,7 @@
     "watch": "run-p 'watch:common' 'watch:api' 'watch:web'",
     "start:api:local": "yarn workspace @ien/api start:dev:local",
     "start:web:local": "yarn workspace @ien/web start",
-    "start:local": "run-p 'watch:common' 'start:api:local' 'start:web:local'",
```

Comment:
What was the issue here?

Reply:
`watch common` means that we are writing codes while keeping the app updated.
I thought `watch` command would be enough for the case and supposed that we run `start:local` after building all apps.